detalhes-pt-br.md

```diff
@@ -33,10 +33,10 @@ In line with Hacktoberfest value #2 (Quantity is fun, quality is key), we have p
 
-## Let's work together to reduce spam
+## Vamos trabalhar juntos para reduzir o spam
 
-- Spammy pull requests can be labeled as "invalid." Maintainers are faced with the majority of spam that occurs during Hacktoberfest, and we dislike spam just as much as you. If you're a maintainer, please label any spammy pull requests submitted to the repositories you maintain as invalid, and close them. Pull requests with this label won't count toward Hacktoberfest.
-- There's a seven-day review window for all pull requests before they count toward completing the challenge. Once a participant has submitted four eligible pull requests (ready-to-review, not drafts), the review window begins. This period gives maintainers time to identify and label spammy pull requests as invalid. If the pull requests are not marked as invalid within that window, they will allow the user to complete the Hacktoberfest challenge. If any of the pull requests are labeled as invalid, the user will return to the pending state until they have four eligible pull requests, at which point the review period will start again.
-- Bad repositories will be excluded. In the past, we've seen many repositories that encourage participants to make simple pull requests – such as adding their name to a file – to quickly gain a pull request toward completing Hacktoberfest. While this may be a learning tool for new contributors, it goes against one of our core values for Hacktoberfest. The quality of pull requests is paramount; quantity comes second. These repositories do not encourage quality contributions and provide an unfair advantage in completing the Hacktoberfest challenge. We've implemented a system to block these repositories, and any pull requests submitted to such repositories will not be counted. 
+- Fazer spam de Pull Request pode ser rotulado como “Invalido”. Os mantenedores lidam com a maior parte do spam que ocorre durante o Hacktoberfest e não gostamos de spam tanto quanto você. Se você for um mantenedor, rotule todas as solicitações de pull request spam enviadas aos repositórios que você mantém como "inválido" ou "spam" e feche-as. Pull Request com esta etiqueta  não contam para o Hacktoberfest.
+- Há uma janela de sete dias para revisão de todas as pull request antes de serem contabilizadas para a conclusão do desafio. Depois que um participante envia quatro pull request elegíveis (prontas para revisão, não rascunhos), a janela de revisão é iniciada. Este período dá aos mantenedores tempo para identificar e rotular as pull request que são spam como inválidas. Se as pull requests não forem marcadas como inválidas nessa janela, elas permitirão que o usuário conclua o desafio do Hacktoberfest. Se qualquer uma das pull request for rotulada como inválida, o usuário retornará ao estado pendente até que tenha quatro pull requests qualificadas, e então período de revisão começará novamente.
```

Comment:
Aqui tbm colocar as palavras em inglês em itálico e na frase "e então período de revisão começará novamente." add "o" "e então o período de revisão começará novamente."

Reply:
Entendi. Tá bom.